tests/puppeteer/.docker/Dockerfile

```diff
@@ -3,23 +3,23 @@
 # Based upon:
 # https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker
 
-FROM node:8-slim
+FROM node:10-slim
 
-RUN  apt-get update \
-     # See https://crbug.com/795759
-     && apt-get install -yq libgconf-2-4 \
-     # Install latest chrome dev package, which installs the necessary libs to
-     # make the bundled version of Chromium that Puppeteer installs work.
-     && apt-get install -y wget --no-install-recommends \
-     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+# Install latest chrome dev package and fonts
+RUN  wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
      && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
      && apt-get update \
-     && apt-get install -y google-chrome-unstable --no-install-recommends \
+     && apt-get install -y google-chrome-unstable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst \
```

Comment:
Are you sure wee need all of theses exotic fonts ? Seems unnecessary to me

Reply:
From puppeteer github : 
> This fonts are used to support major charsets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others) 
I think maybe we will need that for installation tests